apps/vscode/src/refresh-workspace.ts

```diff
@@ -23,55 +18,10 @@ export function registerRefreshWorkspace(context: ExtensionContext) {
 
       getTelemetry().logUsage('misc.refresh-workspace');
 
-      window.withProgress(
-        {
-          location: ProgressLocation.Notification,
-          title: 'Refreshing Workspace',
-          cancellable: false,
-        },
-        async (progress) => {
-          try {
-            const nxlsClient = getNxlsClient();
-            progress.report({ message: 'Stopping nx daemon', increment: 10 });
-            try {
-              await nxlsClient?.sendRequest(NxStopDaemonRequest, undefined);
-            } catch (e) {
-              // errors while stopping the daemon aren't critical
-            }
-            progress.report({ increment: 30 });
+      await getNxlsClient().refreshWorkspace();
```

Comment:
The try-catch-finally isn't used since both `.refreshWorkspace()` and `.restart()` has the try-catch. It's possible in the future that logic added to those methods land outside of their respective try-catch blocks, thus causing `isRefreshing = false` to not execute.